apps/web/src/contexts/globalLayout/GlobalLayoutContext.tsx

```diff
@@ -249,40 +252,50 @@ const GlobalLayoutContextProvider = memo(({ children }: Props) => {
         fadeType={fadeType}
         handleFadeNavbarOnHover={handleFadeNavbarOnHover}
         content={topNavContent}
+        isSidebarPresent={isSidebarPresent}
       />
 
       {isGlobalSidebarEnabled && <GlobalSidebar content={sidebarContent} />}
 
-      <GlobalLayoutStateContext.Provider value={state}>
-        <GlobalLayoutActionsContext.Provider value={actions}>
-          {/*
-           * Fade main page content as the user navigates across routes.
-           * This does not affect the Top Nav or Left Hand Nav.
-           */}
-          <AnimatePresence>
-            <motion.div
-              key={locationKey}
-              initial={{ opacity: 0 }}
-              animate={{ opacity: 1 }}
-              exit={{
-                opacity: 0,
-                transition: { duration: FADE_TRANSITION_TIME_SECONDS },
-              }}
-              transition={{
-                ease: 'easeInOut',
-                delay: NAVIGATION_TRANSITION_TIME_SECONDS,
-                duration: FADE_TRANSITION_TIME_SECONDS,
-              }}
-            >
-              <Suspense fallback={<FullPageLoader />}>{children}</Suspense>
-            </motion.div>
-          </AnimatePresence>
-        </GlobalLayoutActionsContext.Provider>
-      </GlobalLayoutStateContext.Provider>
+      <MainContentWrapper isSidebarPresent={isSidebarPresent}>
```

Comment:
the only change here is wrapping everything in `MainContentWrapper` which just adds extra whitespace on the left of the content to account for when the sidebar is present